src/Tests/dotnet-new.Tests/CommonTemplatesTests.cs

```diff
@@ -168,8 +168,8 @@ public void NuGetConfigPermissions()
 
             var templatesToTest = new[]
             {
-                new { Template = consoleTemplateShortname,  Frameworks = new[] { null, "net6.0", "net7.0" } },
-                new { Template = "classlib", Frameworks = new[] { null, "net6.0", "net7.0", "netstandard2.0", "netstandard2.1" } }
+                new { Template = consoleTemplateShortname,  Frameworks = new[] { null, "net6.0", ToolsetInfo.CurrentTargetFramework } },
```

Comment:
since when the new target framework is added, it needs to be added as new set, this change is not helpful.